fineract-client/src/main/java/org/apache/fineract/client/util/FineractClient.java

```diff
@@ -42,7 +42,7 @@ public static FineractClientBuilder builder() {
         return apiClient.createService(serviceClass);
     }
 
-    public static class FineractClientBuilder {
+    public static final class FineractClientBuilder {
```

Comment:
as above (I think); best to see failing build and check if all 3 or only 1st (I think) need to be fixed?